android/uhabits-core/src/test/java/org/isoron/uhabits/core/models/HabitListTest.java

```diff
@@ -131,56 +130,73 @@ public void testOrdering()
         h4.setName("D Habit");
         h4.setColor(1);
         h4.setPosition(2);
+        HabitList list;
 
-        list.add(h3);
-        list.add(h1);
-        list.add(h4);
-        list.add(h2);
-
+        list = testOrderingSetup(h1, h2, h3, h4);
```

Comment:
Why do we need to recreate the list repeatedly? Does the previous code not work anymore?